src/client/pkg/require/require.go

```diff
@@ -33,6 +33,23 @@ func EqualOneOf(tb testing.TB, expecteds []interface{}, actual interface{}, msgA
 	}
 }
 
+func OneOfEquals(tb testing.TB, expected interface{}, actuals []interface{}, msgAndArgs ...interface{}) {
+	equal := false
+	for _, actual := range actuals {
+		if reflect.DeepEqual(expected, actual) {
+			equal = true
+			break
+		}
+	}
+	if !equal {
+		fatal(
+			tb,
+			msgAndArgs,
+			"Not equal : %#v (expected)\n"+
+				" one of  != %#v (actuals)", expected, actuals)
+	}
+}
+
```

Comment:
No its the opposite. (Maybe a different name would be more clear).

The actual output needs to be one of the expected values.